File: tests/simple-match.js
var BruteForcer = require('../BruteForcer');

var bf = new BruteForcer();

var d1 = +new Date;

if (bf.match('hello')) {
    var diff = +new Date - d1;
    console.log('Successfully matched \'hello\' in', bf.roundcount, 'rounds (' + diff + 'ms)');
}

File: tests/simple-match.py
import sys, os, time
sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
from BruteForcer import BruteForcer

bf = BruteForcer()

d1 = int(round(time.time() * 1000))

if (bf.match('hello')):
    diff = str(int(round(time.time() * 1000)) - d1)
    print "Successfully matched 'hello' in " + str(bf.roundcount) + " rounds (" + diff + "ms)"
